File: source/Fe/src/Eu.EDelivery.AS4.Fe/Runtime/ItemType.cs
using System.Collections.Generic;

namespace Eu.EDelivery.AS4.Fe.Runtime
{
    public class ItemType
    {
        public string Name { get; set; }
        public IEnumerable<Property> Properties { get; set; }
        public string TechnicalName { get; internal set; }
    }
}

File: source/Eu.EDelivery.AS4.Fe/ui/src/app/api/ItemType.ts
/* tslint:disable */
import { FormBuilder, FormGroup, FormArray } from '@angular/forms';
import { Property } from "./Property";

export class ItemType {
	name: string;
	technicalName: string;
	properties: Property[];
	defaultvalue: any;
	description: string;

	static FIELD_name: string = 'name';	
	static FIELD_technicalName: string = 'technicalName';	
	static FIELD_properties: string = 'properties';
	static FIELD_defaultValue: string = 'defaultvalue';
	static FIELD_description: string = 'description';
}
